file-formats/apob/type/zif_aff_apob_v1.intf.abap

```diff
@@ -0,0 +1,62 @@
+INTERFACE zif_aff_apob_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Application Object Description Type</p>
+    "! Attributes of Application Object Description Type
+    BEGIN OF ty_apob_text,
+      "! <p class="shorttext">Language Code</p>
+      "! Language Code
+      "! $required
+      lang    TYPE c LENGTH 1,
+      "! <p class="shorttext">Application Object</p>
+      "! Application Object Name
+      "! $required
+      ao_name TYPE c LENGTH 30,
```

Comment:
Same applies here (and to other names). We even have a type for object names:
```suggestion
      application_object TYPE zif_aff_types_v1=>ty_object_name_30,
```

Reply:
field removed